client/lcd/lcd_test.go

```diff
@@ -522,19 +522,16 @@ func TestBonding(t *testing.T) {
 	resultTx = doBeginRedelegation(t, port, name1, pw, addr, operAddrs[0], operAddrs[1], rdTokens, fees)
 	require.Equal(t, uint32(0), resultTx.Code)
 	tests.WaitForHeight(resultTx.Height+1, port)
-	validator2 := getValidator(t, port, operAddrs[1])
 
 	// query delegations, unbondings and redelegations from validator and delegator
 	delegatorDels = getDelegatorDelegations(t, port, addr)
 	require.Len(t, delegatorDels, 1)
 	require.Equal(t, operAddrs[1], delegatorDels[0].ValidatorAddress)
 
-	// because the second validator never signs during these tests, if this
-	// this test takes a long time to run,  eventually this second validator
-	// will get slashed, meaning that it's exchange rate is no-longer 1-to-1,
-	// hence we utilize the exchange rate in the following test
-	delTokensAfterRedelegation := validator2.TokensFromShares(delegatorDels[0].GetShares())
-	require.Equal(t, rdTokens.ToDec(), delTokensAfterRedelegation)
+	// TODO uncomment once all validators actually sign in the lcd tests
```

Comment:
Please don't comment code out, just remove it. `git ` will preserve history of changes for us anyway.

Reply:
uhhh I don't agree that what you're suggesting is a general good practice, It's going to be extremely difficult to remember and find this later.. right?

Reply:
Are you suggesting adding the commit hash to the TODO comment?

Reply:
> It's going to be extremely difficult to remember and find this late

Well, `git blame` and `git log` are there for a reason :)

> I don't agree that what you're suggesting is a general good practice
> [snip]
> Are you suggesting adding the commit hash to the TODO comment?

No, 'cause after merge&squash commits hashes are lost, and so is your PR's branch history.
I'm just saying: if this is something you want to remove, just remove it. Leaving commented code around is IMO bad practice.

Another option is including your removed snippet in your commit message.